src/Components/Profile/Profile.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { fetchProfile } from "../../actions/profileAction";
+import { NavLink } from "react-router-dom";
+
+export class Profile extends Component {
+	localStorageFunction() {
+		if (this.props.profile.profile) {
+			const data = this.props.profile.profile.user;
+			localStorage.setItem("username", data.profile_user);
+			localStorage.setItem("bio", data.bio);
+			localStorage.setItem("image", data.image);
+		}
+		return "stored";
+	}
+	editButton() {
+		return (
+			<div className="text-center mt-4">
+				<NavLink to={"/EditProfile"}>
+					<button className="btn btn-dark btn-md pull-right" type="submit">
+						Edit
+					</button>
+				</NavLink>
+			</div>
+		);
+	}
+	image() {
+		return (
+			<div className="avatar mx-auto" style={{ "text-align": "center" }}>
```

Comment:
Could you extract inline styles and put them in css file.

Reply:
@shakirandagire I worked on this